test/e2e/clusterdeployment/aws/aws.go

```diff
@@ -40,6 +41,19 @@ func CheckEnv() {
 	})
 }
 
+func PopulateEnvVars(architecture config.Architecture) {
+	GinkgoHelper()
+
+	switch architecture {
+	case config.ArchitectureAmd64:
+		GinkgoT().Setenv(clusterdeployment.EnvVarAWSAMIID, "") // If unset, the default AMI lookup value defined in the template will be used
```

Comment:
Should we make the new envs mandatory, namely, put them into the `CheckEnv` functions of a corresponding provider? This will ensure fail-fast on a local machine e2e tests, as well as potential misuse of test templates. The question is applicable to all of the providers

But maybe it does not make sense since these env variables are not to be customized from a user's inputs